build/repo.targets

```diff
@@ -60,12 +62,24 @@
     <PhysicalScenarios Include="-n Json -m https --kestrelTransport Sockets" />
     <PhysicalScenarios Include="-n Json -m https --webHost HttpSys" />
 
+    <PhysicalScenarios Include="-n Jil" />
+
+    <PhysicalScenarios Include="-n Jil --kestrelTransport Sockets" />
```

Comment:
Would it be sufficient to just test `Jil` and compare to `Json`?  I don't think the other variants of this scenario are worth the cost.

Reply:
Fair enough, we can always add it back later.

Reply:
@DamianEdwards or @sebastienros: Please remove the extra scenarios before merging.  Let's just run `Jil` and `MvcJil` on Physical.